tests/e2e/test_pvc_creation_performance.py

```diff
@@ -165,3 +167,102 @@ def test_multiple_pvc_creation_after_deletion_performance(
         logging.info(
             f"{number_of_pvcs} PVCs creation time took less than a 45 seconds"
         )
+
+    def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
```

Comment:
Its missing the @tier1 or appropriate mark, once that is fixed, you can only run this test from jenkins by providing full path of this test in "tests" box.

Reply:
Please use '@performance' mark for this test.